hack/compare-mock

```diff
@@ -5,6 +5,8 @@ set -x
 
 export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
 
+export KCC_USE_DIRECT_RECONCILERS=SpannerDatabase
```

Comment:
nit: cleanup. (I know this PR is WIP)

Reply:
I am a bit confused on this actually. What is our final / production method for switching a resource to use the "direct" controller instead of the terraform-based controller?

Reply:
To switch a resource to the direct controller, you only need to remove the CRD's ` `tf2crd` label (for DCL based it is `dcl2crd`). If a resource is not yet ready for direct controller but you want to run mock test against the direct controller, you can use the  `KCC_USE_DIRECT_RECONCILERS` flag.
 
In other words, direct controller is the default.